### ISSUE TITLE
Signing issue when opening OpenRefine on MacOS has reappeared with release 3.7 beta2

### ISSUE THREAD

**Issue by trnstlntk** [OpenRefine/OpenRefine#5534]
The signing issue on MacOS was previously fixed (i.e. Mac users needing to right-click the application upon opening it first) but it unfortunately seems to have reappeared with release 3.7 beta2. :(

<img width="372" alt="image" src="https://user-images.githubusercontent.com/9881645/209791907-1810552f-2acf-4b23-a532-a66513573b6d.png">

### Versions
 - Operating System: MacOS 13.1 Ventura
 - OpenRefine: 3.7 beta2

### Additional context
Not sure if this is relevant to the issue at all, but I'm using OR on a very new M2 chip Mac.

**Comment by wetneb** [comment]
This, and [this report](https://forum.openrefine.org/t/or-3-7beta2-or-3-8-20221220-184714-java-included-do-not-work/287) seem to indicate that the packaging pipeline for snapshots and releases could be slightly different. I need to investigate the CI to see where that makes a difference.

**Comment by wetneb** [comment]
I could also reproduce this locally.

**Comment by wetneb** [comment]
The build logs seem to indicate that notarization succeeded:

```
[INFO] --- maven-antrun-plugin:3.1.0:run (generate-dmg) @ packaging ---
[INFO] Executing tasks
[INFO]      [exec] OpenRefine 3.7-beta2/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jli/libjli.dylib: replacing existing signature
[INFO]      [exec] Codesigning JAR file: sqlite-jdbc-3.39.3.0.jar
[INFO]      [exec] OpenRefine.app/Contents/Resources/webapp/extensions/database/module/MOD-INF/lib/tmp/org/sqlite/native/Mac/aarch64/libsqlitejdbc.jnilib: replacing existing signature
[INFO]      [exec] OpenRefine.app/Contents/Resources/webapp/extensions/database/module/MOD-INF/lib/tmp/org/sqlite/native/Mac/aarch64/libsqlitejdbc.jnilib: signed Mach-O thin (arm64) [libsqlitejdbc]
[INFO]      [exec] OpenRefine.app/Contents/Resources/webapp/extensions/database/module/MOD-INF/lib/tmp/org/sqlite/native/Mac/x86_64/libsqlitejdbc.jnilib: replacing existing signature
[INFO]      [exec] Codesigning file libsqlitejdbc.jnilib
[INFO]      [exec] Codesigning file libsqlitejdbc.jnilib
[INFO]      [exec] OpenRefine.app/Contents/Resources/webapp/extensions/database/module/MOD-INF/lib/tmp/org/sqlite/native/Mac/x86_64/libsqlitejdbc.jnilib: signed Mach-O thin (x86_64) [libsqlitejdbc]
[INFO]      [exec] Codesigning JAR file: jython-standalone-2.7.3.jar
[INFO]      [exec] Codesigning file libjansi.jnilib
[INFO]      [exec] OpenRefine.app/Contents/Resources/webapp/extensions/jython/module/MOD-INF/lib/tmp/META-INF/native/osx/libjansi.jnilib: signed Mach-O universal (i386 x86_64) [libjansi]
[INFO]      [exec] OpenRefine.app/Contents/Resources/webapp/extensions/jython/module/MOD-INF/lib/tmp/jni/Darwin/libjffi-1.2.jnilib: replacing existing signature
[INFO]      [exec] Replacing OpenRefine.app/Contents/Resources/webapp/extensions/jython/module/MOD-INF/lib/tmp/jni/Darwin/libjffi-1.2.jnilib by /Users/runner/work/OpenRefine/OpenRefine/packaging/apple_certs/libjffi-1.2.jnilib
[INFO]      [exec] Codesigning file libjffi-1.2.jnilib
[INFO]      [exec] OpenRefine.app/Contents/Resources/webapp/extensions/jython/module/MOD-INF/lib/tmp/jni/Darwin/libjffi-1.2.jnilib: signed Mach-O universal (x86_64 arm64) [libjffi-1]
[INFO]      [exec] Conducting pre-submission checks for openrefine-mac-3.7-beta2.dmg and initiating connection to the Apple notary service...
[INFO]      [exec] Submission ID received
[INFO]      [exec]   id: 22721cbe-4eba-47c2-8cb9-a0033da38e78
[INFO]      [exec] Successfully uploaded file
[INFO]      [exec]   id: 22721cbe-4eba-47c2-8cb9-a0033da38e78
[INFO]      [exec]   path: /Users/runner/work/OpenRefine/OpenRefine/packaging/target/openrefine-mac-3.7-beta2.dmg
[INFO]      [exec] Waiting for processing to complete. Wait timeout is set to 600.0 second(s).
[INFO]      [exec] 
[INFO]      [exec] Current status: In Progress...Current status: In Progress....Current status: In Progress.....Current status: In Progress......Current status: In Progress.......Current status: In Progress........Current status: In Progress.........Current status: In Progress..........Current status: In Progress...........Current status: In Progress............Current status: In Progress.............Current status: In Progress..............Current status: In Progress...............Current status: In Progress................Current status: In Progress.................Current status: In Progress..................Current status: In Progress...................Current status: In Progress....................Current status: In Progress.....................Current status: In Progress......................Current status: In Progress.......................Current status: In Progress........................Current status: In Progress.........................Current status: In Progress..........................Current status: In Progress...........................Current status: In Progress............................Current status: In Progress.............................Current status: In Progress..............................Current status: In Progress...............................Current status: In Progress................................Current status: In Progress.................................Current status: In Progress..................................Current status: Accepted...................................Processing complete  id: 22721cbe-4eba-47c2-8cb9-a0033da38e78
[INFO]      [exec]   status: Accepted
[INFO]      [exec] 
[INFO]      [exec] Processing: /Users/runner/work/OpenRefine/OpenRefine/packaging/target/openrefine-mac-3.7-beta2.dmg
[INFO]      [exec] Processing: /Users/runner/work/OpenRefine/OpenRefine/packaging/target/openrefine-mac-3.7-beta2.dmg
[INFO]      [exec] The staple and validate action worked!
```

**Comment by wetneb** [comment]
The `codesign` utility confirms that it is signed but not notarized correctly, I am not sure why.

```
user@users-Mac-mini other % codesign --verify --test-requirement="=notarized" -vvv OpenRefine.app
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/rmid
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/rmid
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/jaotc
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/jaotc
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/unpack200
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/unpack200
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/jfr
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/jfr
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/rmiregistry
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/rmiregistry
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/jrunscript
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/jrunscript
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/java
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/java
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/pack200
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/pack200
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/keytool
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/keytool
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/jjs
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/jjs
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/NOTICE
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/NOTICE
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/release
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/release
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libnio.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libnio.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libnet.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libnet.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libinstrument.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libinstrument.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libzip.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libzip.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libfreetype.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libfreetype.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/psfontj2d.properties
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/psfontj2d.properties
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/fontconfig.properties.src
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/fontconfig.properties.src
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libsplashscreen.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libsplashscreen.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libmanagement_ext.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libmanagement_ext.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjimage.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjimage.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libj2pkcs11.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libj2pkcs11.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libdt_socket.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libdt_socket.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jvm.cfg
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jvm.cfg
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jli/libjli.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jli/libjli.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/security/cacerts
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/security/cacerts
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/security/blocked.certs
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/security/blocked.certs
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/security/default.policy
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/security/default.policy
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/security/public_suffix_list.dat
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/security/public_suffix_list.dat
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jfr/profile.jfc
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jfr/profile.jfc
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jfr/default.jfc
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jfr/default.jfc
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosxui.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosxui.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosxkrb5.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosxkrb5.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/tzdb.dat
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/tzdb.dat
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjdwp.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjdwp.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libmanagement_agent.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libmanagement_agent.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/librmi.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/librmi.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libsunec.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libsunec.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/server/libjsig.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/server/libjsig.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/server/Xusage.txt
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/server/Xusage.txt
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libawt_lwawt.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libawt_lwawt.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/server/libjvm.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/server/libjvm.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjavajpeg.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjavajpeg.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libmanagement.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libmanagement.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjsound.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjsound.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libj2pcsc.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libj2pcsc.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libmlib_image.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libmlib_image.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjsig.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjsig.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjawt.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjawt.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libprefs.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libprefs.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libfontmanager.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libfontmanager.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jrt-fs.jar
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jrt-fs.jar
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/fontconfig.bfc
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/fontconfig.bfc
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libextnet.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libextnet.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jspawnhelper
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/jspawnhelper
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjaas.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjaas.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosxsecurity.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosxsecurity.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libverify.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libverify.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/liblcms.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/liblcms.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/psfont.properties.ja
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/psfont.properties.ja
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libj2gss.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libj2gss.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjava.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libjava.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/modules
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/modules
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/classlist
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/classlist
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libawt.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libawt.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libunpack.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libunpack.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosxapp.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosxapp.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosx.dylib
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/lib/libosx.dylib
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/logging.properties
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/logging.properties
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/sound.properties
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/sound.properties
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/java.policy
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/java.policy
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/unlimited/default_US_export.policy
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/unlimited/default_US_export.policy
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/java.security
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/java.security
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/unlimited/default_local.policy
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/unlimited/default_local.policy
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/limited/default_US_export.policy
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/limited/default_US_export.policy
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/README.txt
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/README.txt
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/limited/exempt_local.policy
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/limited/exempt_local.policy
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/limited/default_local.policy
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/security/policy/limited/default_local.policy
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/net.properties
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/net.properties
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/management/management.properties
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/management/management.properties
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/management/jmxremote.password.template
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/management/jmxremote.password.template
--prepared:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/management/jmxremote.access
--validated:/Users/user/Downloads/OpenRefine.app/Contents/PlugIns/JRE/Contents/Home/jre/conf/management/jmxremote.access
OpenRefine.app: valid on disk
OpenRefine.app: satisfies its Designated Requirement
test-requirement: code failed to satisfy specified code requirement(s)
```

**Comment by wetneb** [comment]
@trnstlntk if you have a chance to try out 3.7-beta5, it would be great to know if the signing issue is still present there.

**Comment by trnstlntk** [comment]
> @trnstlntk if you have a chance to try out 3.7-beta5, it would be great to know if the signing issue is still present there.

It looks fixed on my side! 👍 Operating System: MacOS 13.1 Ventura, downloaded and opened beta5 without the need to right-click.

**Comment by wetneb** [comment]
Amazing, thanks for the confirmation!
